appveyor/scripts/decryptFilesForSigning.ps1

```diff
@@ -4,7 +4,7 @@ if(!$env:APPVEYOR_PULL_REQUEST_NUMBER) {
 		$errorCode=$LastExitCode
 		Add-AppveyorMessage "Unable to decrypt authenticode certificate"
 	}
-	openssl enc -md md5 -aes-256-cbc -d -pass pass:$env:secure_ssh_pass -in appveyor\ssh_id_rsa.enc -out appveyor\ssh_id_rsa
+	openssl enc -d -md sha256 -aes-256-cbc -pbkdf2 -salt -pass pass:$env:secure_ssh_pass -in appveyor\ssh_id_rsa.enc -out appveyor\ssh_id_rsa

```

Comment:
We may eventually want to put try catch blocks around this and the decrypting of the authenticode certificate above as we no longer expect any output from standard error. However, for this pr at least, it is best to be merged as is so we can firstly ensure the  whole build chain actually runs without errors.